pytorch_lightning/trainer/callback_hook.py

```diff
@@ -39,7 +40,7 @@ def setup(self, model, stage: str):
     def teardown(self, stage: str):
         """Called at the end of fit and test"""
         for callback in self.callbacks:
-            callback.teardown(self, self.get_model(), stage)
+            callback.teardown(self, self.lightning_module, stage)
```

Comment:
do you want to move this outside of the loop to save on unwrapping per callback?

Reply:
I think it's almost not worth it, because unwrapping doesn't really add any overhead since it's just following references, i.e., model.module in case of DDP for example.
I could be wrong, or it may change in the future. Do you see it as a big overhead?